app/unsupported_locale.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta name="description" content="SUMO real time support app for FirefoxOS" />
+  <meta name="viewport" content="width=device-width" />
+
+  <title>Welcome to BuddyUP</title>
+
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/layout.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/headers.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/input_areas.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/buttons.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="css/building_blocks/progress_activity.css" media="screen" />
+
+  <link rel="stylesheet" type="text/css" href="css/fira.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="css/base.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="css/unsupported_locale.css" media="screen" />
+
+  <!-- localization -->
+  <link rel="prefetch" type="application/l10n" href="data/locales.ini" />
+  <link rel="prefetch" type="application/l10n" href="data/date/date.ini" />
+  <script src="js/libs/l10n.js" defer></script>
+  <script src="js/libs/l10n_date.js" defer></script>
+</head>
+<body class="skin-organic">
+<section role="region">
+  <header>
+    <h1>Unsupported Language</h1>
```

Comment:
Putting  these strings in html is going to make localization really hard (read: I have no idea how to do it). Can we put them in a nunjucks or javascript file?